backend/src-common/src/main/java/com/siemens/sw360/datahandler/db/ReleaseRepository.java

```diff
@@ -99,6 +99,14 @@ public ReleaseRepository(DatabaseConnector db, VendorRepository vendorRepository
         return makeSummary(SummaryType.SHORT, queryForIdsByPrefix("byname", name));
     }
 
+    public List<Release> searchByNameAndVersion(String name, String version){
+        List<Release> releasesMatchingName = queryView("byname", name);
```

Comment:
why use `queryView()` and then `makeSummary...` when you could `searchByName` and filter that by version?